### ISSUE TITLE
Show currently played step in stepseq

### ISSUE THREAD

**Issue by mkruselj** [surge-synthesizer/surge#2145]
Sort of a pretty essential thing, especially once Shuffle comes into play... Here's an example from a Kontakt arpeggiator script that has the same facility:

![seqpos](https://user-images.githubusercontent.com/2393720/85009276-7d923f00-b15e-11ea-8e29-dfb43d8bcd69.gif)

**Comment by baconpaul** [comment]
The trick with this (just like with oscillators) is "for which voice"?

**Comment by mkruselj** [comment]
With scene level sequencers is at least easy, but as for per voice, guess I'd say "first played"?

**Comment by VincyZed** [comment]
I thought other plugins used last played?

**Comment by mkruselj** [comment]
Not sure, I think it's first played, if it's last played then the pointer would jump all the freaking time.... But it also might depend on the plugin...

**Comment by baconpaul** [comment]
I'm pushing this out of 1.7.2 since the 'which voice' problem gives us the same problem it does here with all other scope type things.

**Comment by baconpaul** [comment]
Pignent does most recent played fwiw.

**Comment by TheNerdyMusicGuy** [comment]
With this kind of stuff I'd definitely go with the last played voice, as that would be the most useful information, I think. It's also how most plugins do it, I think.

**Comment by VincyZed** [comment]
I agree, I think last played voice is the way to go as well for step-sequencers at voice level. Last voice could also be used to animate other elements in the synth down the line (OSC or LFO displays for instance, if we end up doing that someday). Unless of course we are talking about Scene modulation elements, which in this case it would be first voice of course.
Just a thought :)

**Comment by baconpaul** [comment]
Yup. We can plumb this through for here and mseg (#3289) - carefully - in 1.9
